ops/gcr-cherry-pick.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+INPUT_WHICH_REPO=api
```

Comment:
Do we want these to be parameters to not accidentally cherry pick a version?

Reply:
Or maybe at least have these values not filled out with a comment about what they should be?